pages/about.html

```diff
@@ -5,8 +5,60 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <nav class="nav">
+    <ul>
+      <li><a href="../index.html">Home</a> </li>
+      <li><a href="about.html">About</a> </li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+    </ul>    
+      </nav>
+    <div class = "wrapper">
+    <section>
+      <h2>Who is Roza Ajema?</h2>
+      <p class="paragraph">
+        Meet and greet Roza Ajema. Roza is an inspiring software developer. She has joined Ada developers academy C16 in 2021. Before Ada, she has been working
+        in community development projects in Ethiopia. Roza has one beautiful daughter named Haset. In her free time, Roza enjoyes teaching kids sunday schools, reading stories and spend time with kids. Roza also likes to travel,
+        watch family movies and she cherish her social life. Roza is very grateful for the opportunity Ada gave her to change her career. She chose software development because programming she likes to solve problems to make someone's life easier by using softwares, enhance creativity skills, and 
+        new innovations.
+      </p>
+      </section>
+      <section>
+      <h2> My favourite creatures</h2>
+      <p class="paragraph">I have a special heart to birds. Everytime I see their shape, color and size variation, I am speechless to the creator. Here are some of them in picture.
+        Each colour, each stroke is with purpose which could be survival, camouflage or attraction. What is amazing is that nature is the master at aesthetics plus practicality.
+      </p>
+      <div class="container">
+        <div><img src="images/bird1.jpg" alt="purple">
+          <h3>Gorgeous sprouted hair!</h3>
+        </div>
+        <div><img src="images/bird2.jpg" alt="">
+        <h3>Bride dress with tail!</h3>
+        </div>
+        <div><img src="images/bird3.jpg" alt="">
+        <h3> Sunflower funs! </h3>
+        </div>
+        <div><img src="images/bird4.jpg" alt="">
+        <h3>Peacock</h3>
+        </div>
+        <div><img src="images/bird5.jpg" alt="">
+        <h3>Cutie elbow!</h3>
+        </div>
+        <div><img src="images/bird6.jpg" alt="">
+          <h3>Magnificient purple!</h3>
+        </div>
+      </div>
```

Comment:
This probably would have worked better as a `ul` with `li` tags (since this is a list of things):

```suggestion
      <ul>
        <li>
          <img src="images/bird1.jpg" alt="purple">
          <h3>Gorgeous sprouted hair!</h3>
        </li>
        <li>
          <img src="images/bird2.jpg" alt="">
          <h3>Bride dress with tail!</h3>
        </li>
        <li>
          <img src="images/bird3.jpg" alt="">
          <h3> Sunflower funs! </h3>
        </li>
        <li>
          <img src="images/bird4.jpg" alt="">
          <h3>Peacock</h3>
        </li>
        <li>
          <img src="images/bird5.jpg" alt="">
          <h3>Cutie elbow!</h3>
        </li>
        <li>
          <img src="images/bird6.jpg" alt="">
          <h3>Magnificient purple!</h3>
        </li>
      </ul>
```